Upgrade Knative Serving to v0.8.1 (#5560)
kyma-project/kyma: resources/knative-serving/README.md

This chart includes [knative-serving resources](https://github.com/knative/docs/tree/master/docs/serving) release files.

Included releases:
 * https://github.com/knative/serving/releases/download/v0.8.1/serving.yaml

Kyma-specific changes:
 * The `config-domain` is made configurable by specifying the `.Values.global.domainName` as the helm template.
 * The `knative-serving` Namespace is no longer created. This happens during the installation process.
 * The image versions are changed to use the release tag.

>  * knative-ingress-gateway is now a copy from kyma-gateway.

Reviewer: ```suggestion
 * The `knative-ingress-gateway` is now a copy of `kyma-gateway`.
```

 * The `config-domain` is made configurable by specifying the `.Values.global.domainName` as the helm template.
 * The `knative-serving` Namespace is no longer created. This happens during the installation process.
 * The image versions are changed to use the release tag.

>  * knative-ingress-gateway has been updated

Reviewer 2: we might want to point out that the knative-ingress-gateway is now basically a copy of the kyma-gateway, just in a different namespace
